handbook/marketing/website_style_guide.md

```diff
@@ -7,7 +7,7 @@ Sourcegraph uses the following styles on <a href="https://about.sourcegraph.com"
 
 ## Logos and icons
 
-- [All logo formats](https://drive.google.com/drive/folders/17HFjTFe4q8KLXsu7LoLS_lIgOT1OARiP?usp=sharing)
+- [All logo formats](https://drive.google.com/drive/folders/1lvSh9Q1aFsewAZ-_0tYIIH_nsKPPY0SR)
```

Comment:
@rrhyne  can you please confirm this replacement folder link is correct? I don't see anything IN the folder, which is a bit concerning. After the move, I didn't get any indication that this folder hadn't been moved correctly, so let me know if you see any issues.